Client/Frontend/Strings.swift

```diff
@@ -166,6 +166,7 @@ extension String {
             public static let SectionTitle = MZLocalizedString("FirefoxHome.Pocket.SectionTitle", value: "Thought-Provoking Stories", comment: "This is the title of the Pocket section on Firefox Homepage.", lastUpdated: .v98)
             public static let DiscoverMore = MZLocalizedString("FirefoxHome.Pocket.DiscoverMore", value: "Discover more", comment: "At the end of the Pocket section on the Firefox Homepage, this button appears and indicates tapping it will navigate the user to more Pocket Stories.", lastUpdated: .v98)
             public static let NumberOfMinutes = MZLocalizedString("FirefoxHome.Pocket.Minutes.v99", value: "%d min", comment: "On each Pocket Stories on the Firefox Homepage, this label appears and indicates the number of minutes to read an article. Minutes should be abbreviated due to space constraints. %d represents the number of minutes", lastUpdated: .v99)
+            public static let Sponsored = MZLocalizedString("FirefoxHomepage.Pocket.Sponsored.v102", value: "Sponsored", comment: "This string will show under the description on pocket story, indicating that the story is sponsored.", lastUpdated: .v102)
```

Comment:
I don't think it's going to be released any time soon because there are some pending questions with Sponsor Provider. I added v102 because it was the latest but I'm okay with bumping the version. I should also update the PR name. is v103 ok?

Reply:
PR name should be 102 as that's when the PR name is. THe string name as 102 is fine as well as that's technically when it'll have been added. :)

Reply:
Updated